src/cfnlint/decode/node.py

```diff
@@ -51,6 +54,12 @@ def __deepcopy__(self, memo):
         def __copy__(self):
             return self
 
+        def items(self):
+            """ Provide items so we don't have failures with really bad coded templates """
+            LOGGER.warning('Strings don\'t support items')
```

Comment:
Is there any more context we can give here? Obviously something has gone horribly wrong and it would be nice to provide some info for troubleshooting either the template or the code.  Can we pass the string?"

Reply:
I think so.